handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -796,6 +796,11 @@ public final SSLEngineResult unwrap(
             }
 
             int packetLength = SslUtils.getEncryptedPacketLength(srcs, srcsOffset);
+
+            if (packetLength == SslUtils.NOT_ENCRYPTED) {
```

Comment:
@Scottmitch its the same order in the JDK implementation. So let me keep it like this